### DISCUSSION TITLE
Ne pas afficher le bandeau de consentement initial

### DISCUSSION THREAD

**Discussion by ggoidin** [AmauriC/tarteaucitron.js#626] [Q&A]
Bonjour @AmauriC ,

Je ne parviens pas à trouver une façon "propre" de ne pas afficher le bandeau de gestion des cookies (tarteaucitronAlertBig).
En effet, je pars du principe que si tout est bien configuré et que nous désactivons tous les cookies par défaut, nous sommes en règle avec les recommandations de la CNIL et nous n'avons donc pas besoin de voir le bandeau de consentement initial.
L'activation se ferait donc service tiers par service tiers au bon vouloir du visiteur.

D'ailleurs, la version en place sur le site de la CNIL correspond au fonctionnement attendu.

Y-a-t-il un moyen de cacher ce bandeau dans la version actuelle du plugin ?

Grégory.

**Comment by AmauriC** [comment]
Hello

Cette règle CSS permet de masquer le bandeau de premier niveau:

```
div#tarteaucitronRoot.tarteaucitronBeforeVisible:before,
div#tarteaucitronAlertBig {
    display:none!important;
}
```

**Reply by ggoidin** [reply]
héhé, oui, c'est ce qui est en place sur la CNIL et ce que j'ai mis en attendant.
Je préfère les solutions plus "propre" avec un paramètre en JS mais c'est tout bon :)

Merci pour ta réactivité et ton travail !
